src/main/java/com/google/sps/servlets/PerspectiveRequest.java

```diff
@@ -0,0 +1,20 @@
+package com.google.sps.servlets;
+ 
+import com.google.common.collect.ImmutableMap;
+ 
+/**
+ * This class represents the data needed to send a request
+ * to the Perspective API and will be used to turn the data
+ * into a JSON formatted string.
+ */
+public class PerspectiveRequest {
+    private ImmutableMap<String, String> comment;
+    private ImmutableMap<String, ImmutableMap<String, String>> requestedAttributes;
+ 
+    public PerspectiveRequest(String text) {
+        this.comment = ImmutableMap.of("text", text);
+        // Each attribute maps to a configuration object. No configurations 
+        // are needed, therefore an empty map.
+        this.requestedAttributes = ImmutableMap.of("TOXICITY", ImmutableMap.of());
```

Comment:
Are you going to have multiple attribute types?
If yes: use an enum to store the selection of attribute types you'll want to use. Let me know if you need further clarification on that. and then your map can be Map<Attribute, Map<String, String>>
Example: https://www.w3schools.com/java/java_enums.asp 

If no: Then you don't need a map of maps. You can just store the single ImmutableMap<String, String>, and store "TOXICITY" as a static constant variable.